kornia/geometry/nerf/nerfmm/utils/training_utils.py

```diff
@@ -0,0 +1,65 @@
+import os
+import random
+
+import numpy as np
+import torch
+
+
+def is_DataParallelModel(model):
+    """
+    :param model:   nn.Module or torch.nn.DataParallel(model)
+    :return:        True/False
+    """
+    return hasattr(model, 'module')
+
+
+
+def set_randomness(args):
+    if args.true_rand is False:
+        random.seed(args.rand_seed)
+        np.random.seed(args.rand_seed)
+        torch.manual_seed(args.rand_seed)
+        torch.backends.cudnn.deterministic = True
+        torch.backends.cudnn.benchmark = False
+
+
+def mse2psnr(mse):
```

Comment:
we have `ssim_loss`: https://kornia.readthedocs.io/en/latest/losses.html#kornia.losses.ssim_loss